spec/views/devise/registrations/edit.html.slim_spec.rb

```diff
@@ -16,4 +16,20 @@
 
     expect(rendered).to have_xpath("//form[@autocomplete='off']")
   end
+
+  it 'contains link to enable TOTP' do
+    render
+
+    expect(rendered).to match(users_totp_path)
+  end
+
+  context 'went user is TOTP enabled' do
+    it 'contains link to disable TOTP' do
+      user = build_stubbed(:user, otp_secret_key: '123')
+      @update_user_profile_form = UpdateUserProfileForm.new(user)
+      render
+
+      expect(rendered).to match(disable_totp_path)
```

Comment:
For completeness, can we also check that the other URL is not present? i.e. when the user is TOTP enabled, there should not be a link to enable. This comment applies to the previous test as well.